Bitget.Net/Clients/BitgetRestClient.cs

```diff
@@ -46,6 +48,7 @@ public BitgetRestClient(HttpClient? httpClient, ILoggerFactory? loggerFactory, I
             SpotApiV2 = AddApiClient(new SpotApiV2.BitgetRestClientSpotApi(_logger, httpClient, this, options.Value));
             FuturesApi = AddApiClient(new FuturesApi.BitgetRestClientFuturesApi(_logger, httpClient, this, options.Value));
             FuturesApiV2 = AddApiClient(new FuturesApiV2.BitgetRestClientFuturesApi(_logger, httpClient, this, options.Value));
+            CopyTradingV2 = AddApiClient(new CopyTradingApiV2.BitgetRestClientCopyTradingApi(_logger, httpClient, this, options.Value));
```

Comment:
There is a distinction between the Spot and Futures Copy trading API. This should be called `CopyTradingFuturesV2` so later a `CopyTradingSpotV2` could also be added.